### ISSUE TITLE
Bugs in the section : Page > Enable Scheduling > Change Time 

### ISSUE THREAD

**Issue by soheilkheiri** [dnnsoftware/Dnn.Platform#4529]
## Description of bug
Bugs in the section : Page > Enable Scheduling > Change Time 

## Affected version

* [ ] 10.00.00 alpha build
* [x] 09.09.00 latest supported release

## Affected browser

* [ ] Chrome
* [ ] Firefox
* [ ] Safari
* [ ] Internet Explorer 11
* [ ] Microsoft Edge (Classic)
* [ ] Microsoft Edge Chromium

**Comment by valadas** [comment]
Yeah, there appears to be issues there. Just for the next time, ideally issues should have a bit more details :) But basically:

1. Pick a date and notice it shows up properly
1. Pick a date and then a time and notice the system shows "Invalid Date" in the time picker as well as multiple NaN in the date field.

Screenshot:
![image](https://user-images.githubusercontent.com/6371568/109325236-b0c7ba80-7823-11eb-975c-83dec4561ba1.png)

**Comment by jeremy-farrance** [comment]
I can confirm this problem still exists in DNN 9.11.00 and still needs to be fixed. Daniel's comment above has correct details on how to reproduce.
